x/evmutil/module.go

```diff
@@ -104,7 +116,7 @@ func (am AppModule) Name() string {
 func (am AppModule) Route() sdk.Route { return sdk.Route{} }
 
 // QuerierRoute returns evmutil module's query routing key.
-func (AppModule) QuerierRoute() string { return types.ModuleName }
+func (AppModule) QuerierRoute() string { return types.RouterKey }
```

Comment:
nit: `QuerierRoute` isn't used for modules only using grpc queriers. We could delete the `RouterKey` const and return "" here as [newer sdk modules do](https://github.com/cosmos/cosmos-sdk/blob/35ae2c4c72d4aeb33447d5a7af23ca47f786606e/x/authz/module/module.go#L129-L130)